test/Microsoft.AspNetCore.TestHost.Tests/TestServerTests.cs

```diff
@@ -43,6 +43,13 @@ public void DoesNotCaptureStartupErrorsByDefault()
         }
 
         [Fact]
+        public void TestServerDefaultsToCorrectIpAndPort()
+        {
+            var host = new TestServer(new WebHostBuilder().Configure(app => { }));
+            Assert.Equal(new Uri("http://127.0.0.1:80"), host.BaseAddress);
```

Comment:
We should check the HttpContext.Request.Connection.RemoteIpAddress has the right address.